src/OperationResults.AspNetCore/OperationResults.AspNetCore.csproj

```diff
@@ -30,6 +37,10 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="OperationResultTools" Version="1.0.3" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
```

Comment:
NuGet references should be in alphabetic order.

Reply:
Fixed with 5f25a941423c9bed94670b0c42aad99702351d08